Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM node:8.9.1
+
+RUN yarn global add tslint typescript tslint-react
```

Comment:
# Requested Change
Instead of doing this here, I would recommend adding these as dev dependencies for those of us not using docker for development. This change would automatically propagate to the docker image as well. 

This also has the added bonus of locking down the version. This could be vital for typescript at the very least, because they are known to release breaking changes often :octocat:

Reply:
Silly me, makes a lot of sense :+1: